src/main/java/io/github/jsonSnapshot/SnapshotDataItem.java

```diff
@@ -0,0 +1,67 @@
+package io.github.jsonSnapshot;
+
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import lombok.NonNull;
+import lombok.Value;
+
+@Value
+public class SnapshotDataItem implements Comparable<SnapshotDataItem> {
+
+  private static final int REGEX_FLAGS = Pattern.MULTILINE + Pattern.DOTALL;
+  private static final Pattern REGEX =
+      Pattern.compile("(?<name>[^ =]*) *=+ *(?<data>\\[.*\\])[^\\]]*", REGEX_FLAGS);
+
+  private final String name;
+  private final String data;
+
+  public static SnapshotDataItem ofRawData(@NonNull final String rawDataString)
+      throws IllegalRawDataException {
+    final Matcher matcher = REGEX.matcher(rawDataString);
+    if (!matcher.matches()) {
+      throw new IllegalRawDataException(rawDataString);
+    }
+
+    final String name = matcher.group("name");
+    final String data = matcher.group("data");
+    return SnapshotDataItem.ofNameAndData(trimOrNull(name), trimOrNull(data));
+  }
+
+  private static String trimOrNull(final String str) {
```

Comment:
StringUtils are already on classpath, so `StringUtils#trim` can be used

Reply:
fixed in https://github.com/json-snapshot/json-snapshot.github.io/pull/15/commits/863d2adecadc7d005918e0d152a1a8458d2117fc